examples/networkpolicy/kubernetes-azure-hybrid.json

```diff
@@ -0,0 +1,52 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "azure"
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "linuxpool1",
+        "count": 2,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet",
+        "ipAddressCount": 20,
```

Comment:
during our scrum @colemickens commented that this will have to be configured as a pod restriction.  Otherwise, differing ip address counts will cause intermittent failures.  In this example, one agent from pool2 will be able to take a 21st node, and the second agent from pool1 will not, and it will be not be clear why.